dfanalyzer/graph_visualization/plots.py

```diff
@@ -0,0 +1,131 @@
+# plots for graph visualization
+
+import numpy as np
+import matplotlib.pyplot as plt
+import pandas as pd
+import seaborn as sns
+import dask.dataframe as dd
+
+class GrepIOPlots:
```

Comment:
Also this class seems just general plotting. What is graph specific here? If nothing, move it to the plots.py as new methods.